pages/writing/scaling_support.mdx

```diff
@@ -0,0 +1,808 @@
+    <Heading as="h3" variant="heading30">
+      How we announced it
+    </Heading>
+    <Paragraph>
+      We made the change official by announcing a new support model at an All
+      Hands meeting. It was made alongside many positive announcements. That
+      helped absorb the initial blow, but was also missed by quite a few folks.
+      Keeping this change from unraveling meant we’d need to be persistent. We
+      had to politely, but firmly say no to every request unless it came from
+      the right channel. This also made it a <strong>feelings</strong>{" "}
+      management problem: it was hard to have a team that had once routinely
+      said <em>yes, and</em> to every request, now point to an external URL for
+      any help.
+    </Paragraph>
+  </Column>
+</Grid>
+<Grid gutter="space40">
+  <Column span={4} offset={1}>
+    <Heading as="h3" variant="heading30">
+      Enter the automations
+    </Heading>
+    <Paragraph>
+      We addressed new users first: using Slack’s workflow builder, I built a
+      bot that DM’d anyone who joined our Slack channel for the first time with
+      a greeting, important information, and a link to our support area. While
+      this helped, we still had hundreds of people still active in the channel
+      with a previous social contract.
+    </Paragraph>
+    <Paragraph>
+      To guide our existing customers, we built another bot: this one was a
+      little less discrete. For every new question that showed up in Slack, it
+      posted a response suggesting the user redirect this question to a support
+      channel.
+    </Paragraph>
+    <Paragraph>
+      We sweated the details of this bot's message and tone. It had to be
+      polite, sufficiently firm, and also provide reasons why we were making
+      this change. For example, the bot responded to a slack reaction that's
+      used company wide to indicate better documentation. We also included an
+      escalation path for the truly rare request that needed urgent
+      intervention: while the chances of hitting this button are extremely rare
+      (given that our design system does not directly deploy to customers), it
+      did give customers peace of mind in knowing that a short circuit still
+      existed.
+    </Paragraph>
+    <Paragraph>
+      We launched this about two months ago. Here’s what we’ve observed so far:
+    </Paragraph>
+    <OrderedList>
+      <ListItem>
+        Slack support requests have dropped from 10+ a day to 3 a week. Some of
+        these are redirected to GitHub, but others are managed directly.
+      </ListItem>
+      <ListItem>
+        GitHub discussions did <em>not</em> proportionally increase to 10+ a
+        day. One hypothesis for this is that customers are more willing to try a
+        few things out if the immediacy of Slack is not available.
+      </ListItem>
+    </OrderedList>
+    <Paragraph>
+      But as expected, it's not a rousing success. Customers miss the immediacy
```

Comment:
I feel like you should first more obviously state that the bot was overall successful for us before saying it's not a rousing success.

As it reads now, it feels like you're saying it wasn't successful, or at best, neutral.

Reply:
good point i am v negative